mk/submodule.mk

```diff
@@ -21,7 +21,7 @@ LIBTUV_BOARD := $(BOARD)
 # PIC (Position-Independent-Code) object file
 LIBTUV_OBJS := $(LIBTUV_PATH)/build/$(LIBTUV_PLATFORM)/release/lib/libtuv.o
 
-$(LIBTUV_PATH)/include:
+$(LIBTUV_PATH)/.git:
```

Comment:
I was confused with the transition from `include` to `.git`. Why?

Reply:
I was thinking that the rule is for detecting when to initialize and update the submodule.
Since not every submodule has the same path to enter the directory `include`, I change it to `.git` to unify it.

I wonder whether there is a better suggestion.

Reply:
@jserv should I just revert it?

Reply:
> @jserv should I just revert it?

I think so.

Reply:
Done.